<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M_Set_Times</class>
 <widget class="QDialog" name="M_Set_Times">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>276</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>539</width>
    <height>276</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>539</width>
    <height>276</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>测试设置</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>520</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1</y>
      <width>261</width>
      <height>144</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string>稳定时间</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>  扫描架稳定时间：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_sweepshelf"/>
     </item>
     <item row="0" column="2" rowspan="5">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>多路开关稳定时间：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_multe"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>电子开关稳定时间：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_elesw"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>    波控稳定时间：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_wave"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>    矢网采集时间：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_vna"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>501</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_default">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>3</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>默认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ok">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>3</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>105</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string>测试流程</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>17</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <property name="horizontalSpacing">
      <number>18</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>先配幅再配相：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_m_mp_nums">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>幅相一起配：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_mp_nums">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>144</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string>阵面信息</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_map"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonload">
       <property name="text">
        <string>映射文件</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_array"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonload_2">
       <property name="text">
        <string>阵面文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string>基准查找</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>18</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>13</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <property name="horizontalSpacing">
      <number>16</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>基准范围：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_base_delt_db"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>dB</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>是否修改：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBox_disturb">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
